src/custom/constants/index.ts

```diff
@@ -38,22 +37,6 @@ export const SAFE_APP_CODE = `${DEFAULT_APP_CODE}-SafeApp`
 export const PRODUCTION_URL = 'cowswap.exchange'
 export const BARN_URL = `barn.${PRODUCTION_URL}`
 
-// Allow WALLET_LINK to be activated on mobile
-// since COINBASE_LINK is limited to use only 1 deeplink on mobile
-SUPPORTED_WALLETS_UNISWAP.WALLET_LINK = {
-  ...SUPPORTED_WALLETS_UNISWAP.WALLET_LINK,
-  mobile: true,
-}
-const DISABLED_WALLETS = /^(?:Portis|COINBASE_LINK)$/i
```

Comment:
We might want to keep this

Reply:
So this was basically only used here https://github.com/cowprotocol/cowswap/blob/619b1bec31b44fa741f97d7fb1bb00b549b47ce1/src/custom/constants/index.ts#L51 to filter the Uniswap defined connector options, but now, this is changed and the SUPPORTED_WALLETS const doesn't even exists and each connection option is defined separately in its own component so I think if they decide to add those wallets we will notice it